app/components/primer/beta/blankslate.pcss

```diff
@@ -1,12 +1,20 @@
 /* blankslate */
 
+.blankslate-container {
+  container-type: inline-size;
+}
+
 .blankslate {
+  --blankslate-outer-padding-block: var(--base-size-32);
```

Comment:
I noticed padding has been removed from number of our preview snapshots, I assume as a result of this change. Is this going to be ok for all the existing blankslates in dotcom?

Reply:
@langermank - could this be because PVC is using a version of PCSS or Primitives that doesn't have `--base-size-32` defined?

When I tested in Lookbook, everything looked correct 🤔 

Current versions:
```
"@primer/css": "21.0.2",
"@primer/primitives": "^7.14.0",
```

Reply:
Hmm I'm not following 🤔 I think the snapshots look accurate to the changes that were made (mostly see a font-size change). Those base sizing tokens are available and safe to use 👍

Reply:
This is what I see in a bunch of our blankslate snapshots:

<img width="904" alt="A screenshot from this PR showing the visual differences between one of our blankslate previews. The screenshot is itself composed of two side-by-side screenshots of the preview in question. The left-hand screenshot is labeled 'Deleted' in red text, while the right-hand side screenshot is labeled 'Added' in green text. The right-hand ('Added') screenshot is visibly shorter, which can be seen by noticing the top and bottom of the two screenshots are not flush with each other." src="https://github.com/primer/view_components/assets/575280/32f59c4f-4fad-4b7c-bc3d-7146e4aceca6">

Notice how the "Added" one is slightly shorter. Maybe not a big deal, just checking ✅

Reply:
Ok, consensus is the spacing issues I'm concerned about are attributable to the changes in font size 👍